examples/agent-sandbox-rl/agent_sandbox_rl/config.py

```
@@ -121,10 +121,9 @@ class FleetConfig(BaseModel):
   ready_timeout: int = 900
   # Stage the warm fill in waves of <= this many sandbox creates in flight,
   # waiting for each wave to reach Ready before the next. Bounds the controller's
-  # concurrent create burst (Σ pools×replicas) so a large/deep warm can't trip the
-  # SandboxWarmPool over-creation race (#1215). 0 = warm everything at once (old
-  # behavior). The safe value scales inversely with controller warm-pool worker
-  # concurrency; 1000 is calibrated for a high-worker controller.
+  # concurrent create burst (Σ pools×replicas). On controllers <= v0.5.3 this also
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)